src/parse.d

```diff
@@ -231,6 +231,20 @@ struct PrefixAttributes
     const(char)* comment;
 }
 
+/*****************************
+ * Destructively extract storage class from pAttrs.
+ */
+private StorageClass getStorageClass(PrefixAttributes* pAttrs)
```

Comment:
'get' sounds it has no side effect... 'extract' might be better like as the comment text?